src/App.js

```diff
@@ -1,23 +1,17 @@
 import React from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
 
-import logo from "./logo.svg";
-
-import "./App.css";
-
-function App() {
+import Login from "./pages/Login/index";
+import PATH from "./constants/paths";
```

Comment:
Eliminar esta referencia

```suggestion
import PATH from "./constants/paths";
```